chapter03/qli/src/test/resources/test.xml

```diff
@@ -1,14 +1,22 @@
 <beans xmlns="http://www.springframework.org/schema/beans"
 	xmlns:context="http://www.springframework.org/schema/context"
-	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:aop="http://www.springframework.org/schema/aop"
 	xsi:schemaLocation="
 http://www.springframework.org/schema/beans
-http://www.springframework.org/schema/beans/spring-beans-3.0.xsd
+http://www.springframework.org/schema/beans/spring-beans.xsd
 http://www.springframework.org/schema/context
-http://www.springframework.org/schema/context/spring-context-3.0.xsd">
+http://www.springframework.org/schema/context/spring-context.xsd
+http://www.springframework.org/schema/aop
+http://www.springframework.org/schema/aop/spring-aop.xsd">
 
+	<context:annotation-config />
 	<context:component-scan base-package="spring.training.beanConfig" />
-	
+
 	<bean id="service" class="spring.training.service.SpringBankService" />
+	
+	
+    <aop:aspectj-autoproxy proxy-target-class="true"/>
```

Comment:
I see some difference between my code and yours  in the xml name space- schemalocation. I used the specific version. Do you think, that make a difference?